packages/recurring-payments-job/src/services/govuk-pay-service.js

```diff
@@ -0,0 +1,5 @@
+import { govUkPayApi } from '@defra-fish/connectors-lib'
+
+export const sendPayment = async preparedPayment => {
+  govUkPayApi.createPayment(preparedPayment, true)
+}
```

Comment:
This doesn't need to be async - no promise is returned, so it can be a sync function